content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
+
+<Callout type="info" title="Note">
+  A "fork" is a copy of a software project that is modified in some way.\
+  They typically share the same codebase but have modified (or additional) features and optimizations.\
+  An example of "forks" include: Bukkit -> CraftBukkit -> Spigot -> Paper -> Pufferfish -> etc.
+</Callout>
+
+## Vanilla
+
+Vanilla is the official server software for Minecraft. As the name implies, it is the most basic server software available and has no modifications whatsoever.\
+It is ideal for those who wish to host a small server for a couple of friends or family, **without** altering any mechanics.\
+This server software is **not** recommended for larger servers.
+
+<Card href="https://jar.setup.md/download/vanilla/latest" title="Download">
+  Download a copy of the latest Vanilla server software via jar.setup.md.
+</Card>
+
+## Paper
+
+[Paper](https://papermc.io/) is arguably the most widely used server software today. It is a fork of the Spigot project (with its own API), featuring **numerous optimizations** alongside numerous bug fixes, making it the main choice for many aspiring server owners.
+
+Additionally, it has support for plugins which bring new features, QOL improvements and other enhancements to the game.\
+You can find plugins on platforms such as [SpigotMC](https://www.spigotmc.org/resources/categories/spigot.4/), [Modrinth](https://modrinth.com/plugins), [Hangar](https://hangar.papermc.io/), etc.
+
+<Callout type="warn" title="Warning">
+  While Paper features many optimizations and features, many of these affect vanilla gameplay mechanics.\
+  This makes it unsuitable for servers that wish to maintain a vanilla experience or those who make complex farms that utilize intricate vanilla mechanics.
+</Callout>
+
+<Card href="https://jar.setup.md/download/paper/latest/latest" title="Download">
+  Download a copy of the latest PaperMC server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Pufferfish (recommended)
```

Comment:
I have some issues with recommending Pufferfish so explicitly. I think the smartest and safest choice for the community is to recommend Paper, while highlighting its weakness (intentionally broken quirk-based vanilla farms) and suggesting alternatives to suit specific usecases.

I think it's perfectly okay to espouse the value of Paper forks and alternatives, as they certainly do each have their own upsides and downsides, but Paper being the default is smart. We can basically guarantee it will always have substantial development power behind it, so it will receive fixes fast, will have a fast release cycle, and will have reliable documentation and community support. Plus we reduce the debugging surface for newbies with questions by consolidating many/most of them to the same software. Additionally, of all of the various open source projects, Paper is least likely to disappear suddenly.

Reply:
Oh absolutely, that’s a very good point.
I will also make sure to state that forks will take longer to update due to it having to pull from upstream and update their own patches

Reply:
Resolved in [15f25cb](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/15f25cb1bf1de36cd02082cd0d809ecd180d70ef)